Selenium_cource/home_work.rb

```diff
@@ -18,294 +20,297 @@
+            @browser.find_element(id: "user_password_confirmation").send_key password
+            @browser.find_element(id: "user_firstname").send_key name
+            @browser.find_element(id: "user_lastname").send_key "last_name_"+login
+            @browser.find_element(id: "user_mail").send_key login + "@usermail.com"
+            Selenium::WebDriver::Support::Select.new(@browser.find_element(:id, "user_language")).
                 select_by(:value, "en")
-            browser.find_element(name: "commit").click
+            @browser.find_element(name: "commit").click
             sleep 2
-
-      #Check registration
-            if (browser.find_element(id: "flash_notice").text) == 'Ваша учётная запись активирована. Вы можете войти.'
-
-              puts 'User registration = OK'
-
-            else
-
-              puts 'Registration failed'
-
-            end
-            #SAVE ID witch use to find user
-              id_user = browser.find_element(class: "active").attribute("href")[-5,5]
-                  puts "Id of new user = " + id_user
+      fail 'Did not meet expected text' unless message_is('Ваша учётная запись активирована. Вы можете войти.')
+            puts "Create new user = OK"
     end
+#Get text for Verification for "flash_notice"
 
-#Register_new_user_method
-
-    def self.create_new_account(browser, new_user_login, password, name_new_user)
-
-
-          #fill fields
-          browser.find_element(class: "register").click
-          browser.find_element(id: "user_login").send_key new_user_login
-          browser.find_element(id: "user_password").send_key password
-          browser.find_element(id: "user_password_confirmation").send_key password
-          browser.find_element(id: "user_firstname").send_key name_new_user
-          browser.find_element(id: "user_lastname").send_key name_new_user
-          browser.find_element(id: "user_mail").send_key new_user_login + "@usermail.com"
-          Selenium::WebDriver::Support::Select.new(browser.find_element(:id, "user_language")).
-              select_by(:value, "en")
-          browser.find_element(name: "commit").click
-        sleep 2
-
-          #Check registration
-            if (browser.find_element(id: "flash_notice").text) == 'Ваша учётная запись активирована. Вы можете войти.'
-
-              puts 'New user registration = OK'
-
-            else
+                def message_is(message)
+                  @browser.find_element(id: 'flash_notice').text.include? message
+                end
 
-              puts 'Registration failed'
 
-            end
-          #SAVE ID witch use to find user
-          id_user = browser.find_element(class: "active").attribute("href")[-5,5]
-          puts "Id of new user = " + id_user
-
-    end
-
-#ID User Method
-        def get_user_id(browser)
-          id_user = browser.find_element(class: "active").attribute("href")[-5, 5]
+#Get ID User Method
+        def get_user_id
```

Comment:
you can remember that method in default case returns result of last code line run
so inside of method you can leave just
@browser.find_element(class: "active").attribute("href")[-5, 5]
without any variables and return statement